controllers/cartController.js

```diff
@@ -0,0 +1,17 @@
+const cartService = require('../services/cartService');
+const { catchAsync } = require('../utils/error');
+
+const insertCart = catchAsync(async (req, res) => {
+  const { productId, quantity } = req.body;
+  const userId = await req.userId;
+  if (!userId || !productId || !quantity) {
+    const error = new Error('KEY_ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
```

Comment:
1. 원종님 코드가 req.userId에서 req.user로 수정 될 예정입니다. 따라서 userId 가져오는 코드 수정이 필요합니다.
2. if 구문 앞뒤로 공백을 추가하여 가독성을 올려주세요.
```suggestion
  const { productId, quantity } = req.body;
  const userId = await req.user.id;
  
  if (!userId || !productId || !quantity) {
    const error = new Error('KEY_ERROR');
    error.statusCode = 400;
    throw error;
  }
```